cmd/shim/main.go

```diff
@@ -242,8 +243,14 @@ func shim() (returnExitCode int) {
 			stderrRedirect = stderrRedirectFile
 		}
 
-		outWriter := io.MultiWriter(stdoutFile, stdoutRedirect, os.Stdout)
-		errWriter := io.MultiWriter(stderrFile, stderrRedirect, os.Stderr)
+		combinedFile, err := os.Create(combinedPath)
+		if err != nil {
+			panic(err)
+		}
+		defer combinedFile.Close()
+
+		outWriter := io.MultiWriter(stdoutFile, stdoutRedirect, combinedFile, os.Stdout)
+		errWriter := io.MultiWriter(stderrFile, stderrRedirect, combinedFile, os.Stderr)
```

Comment:
This is fine, but just wanted to note - because the secret scrubber can introduce an amount of latency (if we're halfway through a secret), we might end up shuffling things around.

I'm not quite sure the general semantics of Stdout/Stderr merging though - I know shells make stdout/stderr the same file descriptor, but we kinda can't do that, since we wanna preserve being able to get them separately.